odl/space/ntuples.py

```diff
@@ -499,19 +501,19 @@ def fallback_copy(x1, x2, n):
         _lincomb(a+b, x1, 0, x1, out, dtype)
     elif out is x1 and out is x2:
         # All the vectors are aligned -> out = (a+b)*out
-        scal(a+b, out.data, out.size)
+        scal(a+b, out.data, native(out.size))
```

Comment:
Having casts everywhere he is kindof against the idea of this beeing optimized. Not 100% convinced.